gradle.properties

```diff
@@ -21,28 +21,8 @@ org.gradle.configureondemand=true
 # When set to true, Gradle will reuse task outputs from any previous build, when possible.
 org.gradle.caching=true
 
-# Enable Kapt Incremental annotation processing requeste
-kapt.incremental.apt=true
-
-# Decrease gradle builds time
-kapt.use.worker.api=true
-
-# turn off AP discovery in compile path, and therefore turn on Compile Avoidance
-kapt.include.compile.classpath=false
-
 # AndroidX package structure to make it clearer which packages are bundled with the
 # Android operating system, and which are packaged with your app's APK
 # https://developer.android.com/topic/libraries/support-library/androidx-rn
 # Android plugin uses the appropriate AndroidX library instead of a Support Library.
-android.useAndroidX=true
-
-# Automatically convert third-party libraries to use AndroidX
-android.enableJetifier=true
-
-# Once you have globally enabled the Daemon in this way, all your builds will take advantage of the
-# speed boost, regardless of the version of Gradle a particular build uses.
-org.gradle.daemon=true
-
-# Run with file system watching enabled
-# https://docs.gradle.org/current/userguide/gradle_daemon.html#sec:daemon_watch_fs
-org.gradle.vfs.watch=true
```

Comment:
Could you explain why we don't need this anymore or why was not been used? Thanks!

Reply:
We don't need it because it's enabled by default from Gradle 7.0 and later and we are using Gradle 7.0.2.